index.html

```diff
@@ -20,10 +20,240 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+   <!-- Our page wrapper and main flex container -->
+  <div class="page">
+    <header>
+      <div class="header-banner">
+        <nav class="header-banner-nav">
+          <ul>
+            <li><a href="#">ABOUT</a></li>
+            <li><a href="#">BOOK</a></li>
+          </ul>
+        </nav>
+        <div class="header-banner-title">
+          <h1>Luxe Hair and Salon</h1>
+          <img src="logo.png" width="40" />
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+        </div>
 
-    -->
+        <nav class="header-banner-nav2">
+          <ul>
+            <li><a href="#">OUR TEAM</a></li>
+            <li><a href="#">GALLERY</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <hero>
+      <div class="hero-section">
+
+        <div class="hero-content">
+          <div class="hero-content-title">
+            <h1>HIGHLIGHT YOUR INNER BEAUTY</h1>
+          </div>
+          <div class="hero-content-text">
+            <p>Say goodbye to split ends and hello to healthy hair! Pamper yourself and book an appointment with Luxe
+              Hair
+              and Salon.</p>
+          </div>
+
+          <section>
+            <ul class="hero-content-box">
+              <li>
+                <section>
+                  <img src="clock.png" width="30">
+                  <h3>OPENING TIMES</h3>
+                  <p>Tue-Sat 8am-7pm Sun. 8am-5pm</p>
+                </section>
+              </li>
+              <li>
+                <section>
+                  <img src="house.png" width="30">
+                  <h3>LOCATION</h3>
+                  <p>Ottawa, ON</p>
+                </section>
+              </li>
+              <li>
+                <section>
+                  <img src="calendar.png" width="30">
+                  <h3>BOOK NOW</h3>
+                  <p>Book Online or call (613)-999-9999</p>
+                </section>
+              </li>
+            </ul>
+          </section>
+        </div>
+
+        <img class="hero-image" src="hair.png" width="960">
+      </div>
+  </div>
+  </hero>
+
+
+  <main>
+    <!--Sub Section 1: OUR PHILOSOPHY-->
+    <div class="sub-section-1">
+      <div class="section-1-title">
+        <p>OUR PHILOSOPHY</p>
+      </div>
+      <div class="section-1-content">
+        <p>At first impression, we're no different from your average salon: buzzing with blow dryers and conversations
```

Comment:
Couple things here, 

Your `main` area should have an `h1` to start. You can hide it using CSS if you don't want it to appear visually. 

And the "OUR PHILOSOPHY" should also be a heading since it appears more like a heading than a regular paragraph.